api/src/main/java/io/opencensus/resource/Resource.java

```diff
@@ -99,14 +104,34 @@ public static Resource create() {
    *     ASCII string or exceed {@link #MAX_LENGTH} characters.
    * @since 0.18
    */
-  public static Resource create(@Nullable String type, Map<String, String> labels) {
+  // TODO(mayurkale): Use this method while creating resource information from GCP, AWS etc.
```

Comment:
This comment it is not necessary now that you explain where do you need this.

Reply:
Done.